extensions/panache/hibernate-reactive-panache-common/runtime/src/main/java/io/quarkus/hibernate/reactive/panache/common/runtime/AbstractJpaOperations.java

```diff
@@ -396,10 +423,12 @@ public static <T extends Mutiny.AbstractQuery> T bindParameters(T query, Map<Str
     }
 
     public static Uni<Integer> executeUpdate(String query, Object... params) {
+        // TODO Luca not sure about this
```

Comment:
Same thing happens in the ORM version. The day someone wants to support multiple PUs here, we'll ask for what PU to use in `Panache.executeUpdate` where this is called.